### PR TITLE
Add abstract for Collaborations Workshop 2022 mini-workshop

### PR DESCRIPTION
**Pull request by sgibson91**
relates to #33 

This PR adds an abstract to CollabW22 to run a pilot of the survey discussed in #32 as a mini-workshop

### Deadline for submission is 4th February 2022

Please submit your review at your earliest convenience before that date

### CODE REVIEW

File: talks/abstracts/CollabW2022.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+The International Interactive Computing Collaboration (2i2c)
+
+## Session Collaborators
+
+_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._
+
+- Meag Doherty, NIH
+- Min Ragan-Kelley, Simula
+- Achintya Rao, The Alan Turing Institute
```
Comment (sgibson91) [inline talks/abstracts/CollabW2022.md:23]: If you plan on attending CollabW22, please add your preferred email address.

I do recommend attending as it is a wonderful event, but it is not critical 🙂

File: talks/abstracts/CollabW2022.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Mini-workshop or demo session title
+
+Crowdsourcing Community Practices for Defining Reproducible Computational Environments in the Cloud
+
+## Mini-workshop or demo session abstract
+
+_If your session is accepted this abstract will be made public on the CW22 website, so please write this as a public description containing 200-400 words._
+
+_Current word count: 230_
```
Comment (sgibson91) [inline talks/abstracts/CollabW2022.md:33]: Max limit is 400, so we have space to play here!

File: talks/abstracts/CollabW2022.md (hunk 1 above)
Comment (RaoOfPhysics) [reply talks/abstracts/CollabW2022.md:23]: My Turing address is <arao@turing.ac.uk>. :)

File: talks/abstracts/CollabW2022.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._
+
+- Meag Doherty, NIH
+- Min Ragan-Kelley, Simula
+- Achintya Rao, The Alan Turing Institute
+
+## Mini-workshop or demo session title
+
+Crowdsourcing Community Practices for Defining Reproducible Computational Environments in the Cloud
```
Comment (RaoOfPhysics) [inline talks/abstracts/CollabW2022.md:27]: Could potentially drop “Defining”. Maybe change “Crowdsourcing” to “Contribute” to make it an action for the reader?

> Contribute community practices for reproducible computational environments in the cloud

Hmm, sounds a bit flat, but maybe we can iterate on this if you like.

File: talks/abstracts/CollabW2022.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+## Mini-workshop or demo session abstract
+
+_If your session is accepted this abstract will be made public on the CW22 website, so please write this as a public description containing 200-400 words._
+
+_Current word count: 230_
+
+mybinder.org is a very popular tool for hosting interactive and reproducible computational environments in the cloud.
```
Comment (RaoOfPhysics) [inline talks/abstracts/CollabW2022.md:35]: Maybe make the first line a bit punchier?

```markdown
mybinder.org makes your reproducible notebooks interactive via the cloud.
The ethos of this popular service for hosting interactive computational environments is…
```

File: talks/abstracts/CollabW2022.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+The ethos of this project, and those that power it, is _"to automate existing community best practices. If a community isn't doing something in the way we think they are, we shouldn't implement what we think."_
+While Binder supports a range of languages across the open source software ecosystem (and beyond!), Binder itself is primarily a Python package that is maintained by Python developers.
+And so keeping up to date with other communities and ensuring the project doesn't fall too far behind in terms of best practice is a large effort that often happens outside of the maintainer's network.
+
+For this mini-workshop, we would like to invite a diverse range of people working across various languages to participate in an interactive survey pilot and facilitated discussion around the best practices they currently use for defining reproducible computational environments.
+The aim of the session will be to better understand the pain points of using a service like Binder, and how Binder's assumptions may or may not be mismatched with various community best practices.
```
Comment (RaoOfPhysics) [inline talks/abstracts/CollabW2022.md:41]: “mismatched” &rarr; “aligned”

File: talks/abstracts/CollabW2022.md (hunk 4 above)
Comment (sgibson91) [reply talks/abstracts/CollabW2022.md:35]: I generally like it, I'm only going to push back on explicitly calling out notebooks. We have enough trouble telling people you can use scripts and a terminal in binder as it is 😉  So something like this maybe?

```
mybinder.org transforms your static code into a reproducible and interactive environment running in the cloud.
```

File: talks/abstracts/CollabW2022.md (hunk 5 above)
Comment (sgibson91) [reply talks/abstracts/CollabW2022.md:41]:
```suggestion
The aim of the session will be to better understand the pain points of using a service like Binder, and how Binder's assumptions may or may not be aligned with various community best practices.
```

File: talks/abstracts/CollabW2022.md (hunk 1 above)
Comment (sgibson91) [reply talks/abstracts/CollabW2022.md:23]:
```suggestion
## Session Collaborators

_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._

- Meag Doherty, NIH
- Min Ragan-Kelley, Simula
- Achintya Rao, The Alan Turing Institute, arao@turing.ac.uk
```

**Comment by sgibson91** [comment]
After a chat with @meagdoh today, I rewrote this. Specifically, I try not to mention Binder too much so as to not put off the non-Binder folk from attending. I've also tried to include stronger statements of the problem and the stakes if we do nothing. I'm now at 423 words, so a little bit of fat can be trimmed. Let me know what you think!

**Review by minrk — COMMENTED** [review]
I don't know if there's room, but a point that might be worth getting across surrounding our "automating existing best practices" mission is:

- following these practices is beneficial to practitioners, whether they use a tool like Binder or not
- A collection of cross-community best practices is useful beyond Binder
  - The same information is useful to  for _anyone_ building such a tool
  - It's useful for individuals who want to learn about best practices from their own or other communities, which are typically spread all over the internet

This is implied in the problem statement in the first paragraph, but it might be worth rephrasing the end a little bit to signal that the impact can be broader than just feedback to Binder itself

File: talks/abstracts/CollabW2022.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+To begin to address these concerns, we need to open a dialogue with communities and collect feedback to guide development and identify awareness gaps in how the tool is being used and is expected to work.
+
+For this mini-workshop, we would like to invite a diverse range of people working across various languages to participate in an interactive survey pilot and facilitated discussion around the best practices they currently use for defining reproducible computational environments.
+The aim of the session will be to better understand the pain points of using a cloud-based environment service, and how the assumptions such a service makes may or may not align with various community best practices.
+As a popular tool in this area, we will be using mybinder.org as an example of such a service, though prior knowledge is not a requirement, and we hope to build strong connections with active users across communities to provide specific and useful feedback to the Binder team.
```
[the remark this answers is not in the document]
Comment (sgibson91) [reply talks/abstracts/CollabW2022.md:49]: Thanks, this is great!!

**Comment by sgibson91** [comment]
Submitted! Thanks everyone!

**Pull request merged**